src/pages/Login/sunyoung/Login.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Nav from '../../../components/Nav';
+import '../sunyoung/Login.scss';
```

Comment:
```suggestion
import './Login.scss';
```

한 번 나가지 않고도 쓸 수 있을 것 같아요 👀